fake-ipa/fake_ipa/inspector.py

```diff
@@ -30,12 +30,13 @@ def inspect(system, inspection_callback_url, verify, cert, logger):
                     "lldp": None,
                     "product": "0x0001",
                     "vendor": "0x1af4",
-                    "name": "eth1",
+                    "name": nic.get("name", f"enp1s{i}"),
                     "has_carrier": True,
-                    "ipv4_address": system.get("nics")[0]["ip"] or None,
+                    "ipv4_address": nic.get("ip"),
                     "client_id": None,
-                    "mac_address": system.get("nics")[0]["mac"],
+                    "mac_address": nic["mac"],
```

Comment:
Why not `nic.get("mac")` the same as ip/name?

Reply:
I want to fail with error in inspection if it could not get mac

Reply:
OK let's make it None if it does not exist and let it fail somewhere else if no mac

Reply:
It already fails at L26 if the nic is not OK for first NIC.

Nics from 1 to x would then pass inspection with None values, so if the mac is the ONLY meaningful value nic has to have, then its OK to leave as is.

Reply:
Yeah let's fail if boot_interface doesnt have a mac else they other can be a dummy values where it is still ok to progress if the mac is missing